File: algorithm/Python3_version/leetcode5453.py
class Solution:
    def getLastMoment(self, n: int, left: List[int], right: List[int]) -> int:
        t = 0
        for pos in left:
            t = max(t, pos)
        for pos in right:
            t = max(t, n-pos)
        return t

File: algorithm/leetcode5453.cpp
class Solution {
public:
    int getLastMoment(int n, vector<int>& left, vector<int>& right) {
    	int t = 0;
    	for(int pos : left)
    		t = max(t, pos);
    	for(int pos : right)
    		t = max(t, n-pos);
    	return t;
    }
};